### PR TITLE
fix(es): remove duplicate headings

### PR DESCRIPTION
**Pull request by yin1999**
### Description

remove duplicate headings. Since I don't speak Spanish, please feel free to correct my mistakes.

### Motivation

Prepare for enabling [the `no-duplicate-heading` markdown-lint rule](https://github.com/mdn/translated-content/blob/d13c47b2b819b24e7623bb55ad5f24606c537d87/.markdownlint.jsonc#L20-L22).
Part of #17582

### CODE REVIEW

File: files/es/web/html/element/input/range/index.md [hunk 1]
```diff
@@ -199,44 +199,49 @@ Captura de pantalla
...
+input[type="range"] {
+  appearance: slider-vertical;
+}
+```
 
-{{EmbedLiveSample("Orientation_sample2", 200, 200, "orientation_sample2.png")}}
+{{EmbedLiveSample("Usando la propiedad de appearance", 200, 200)}}
 
-**En la actualidad, ninguno de los navegadores principales soporta la creación de inputs range usando este CSS, incluso a perar de que la especificación recomienda que lo hagan.**
+#### Control de rango vertical
```
Comment (yin1999) [inline files/es/web/html/element/input/range/index.md:240]: The following content has not been updated (We can update this content in the future)

File: files/es/learn/css/css_layout/floats/index.md [hunk 2]
```diff
@@ -328,7 +314,7 @@ body {
 }
 ```
 
-{{ EmbedLiveSample('Float_3', '100%', 600) }}
+{{ EmbedLiveSample('Limpiando floats', '100%', 600) }}
```
Comment (JuanVqz) [inline files/es/learn/css/css_layout/floats/index.md:317]: floats is English, so, I think there should be flotadores in plural
```suggestion
{{ EmbedLiveSample('Limpiando flotadores', '100%', 600) }}
```

Comment (yin1999) [reply files/es/learn/css/css_layout/floats/index.md:317]: We should use the [leading heading/ID of the group of codes](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Page_structures/Live_samples#grouping_code_blocks). For this part, we should refer [this heading](https://github.com/mdn/translated-content/pull/17617/files#diff-6e214aac65d71ec2a87b5e178d4e33d1065221a2eed56a84cd3e99a3b6e14584R247). So I used "Limpiando floats". If you would like to fix this, please also correct the content of the above heading :)

Comment (JuanVqz) [reply files/es/learn/css/css_layout/floats/index.md:317]: Thanks for clarifying it

**Pull request merged**